lua/gitlab/reviewer/diffview.lua

```diff
@@ -10,8 +10,19 @@ local M = {
 }
 
 M.open = function()
-  vim.api.nvim_command(string.format("DiffviewOpen %s", state.INFO.target_branch))
+  local diff_refs = state.INFO.diff_refs
+  if diff_refs == nil then
+    u.notify("Gitlab did not provide diff refs required to review this MR", vim.log.levels.ERROR)
+    return
+  end
+
+  vim.api.nvim_command(string.format("DiffviewOpen %s..%s", diff_refs.start_sha, diff_refs.head_sha))
```

Comment:
Now rather than opening up with the two branch names we open up with the specific commits provided from Gitlab, after making sure they are available locally by pulling down the latest via `git fetch origin`

Reply:
@harrisoncramer I noticed that by doing this we loose a lot of goodies of neovim like LSP :thinking:.

Maybe it would be better to just do
```lua
vim.api.nvim_command(string.format("DiffviewOpen %s", diff_refs.base_sha))
```
and notify user if `diff_refs.head_sha` is not same as `git rev-parse HEAD` ? What do you think ?